install/mysql/glpi-empty.sql

```diff
@@ -9970,4 +9970,21 @@ CREATE TABLE `glpi_assets_assettypes` (
   KEY `date_creation` (`date_creation`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 
+DROP TABLE IF EXISTS `glpi_assets_customfields`;
+CREATE TABLE `glpi_assets_customfields` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `assets_assetdefinitions_id` int unsigned NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `label` varchar(255) NOT NULL,
+  `type` varchar(255) NOT NULL,
+  `flags` tinyint NOT NULL DEFAULT 0,
+  `itemtype` VARCHAR(255) NULL DEFAULT NULL,
+  `default_value` text,
+  `allowed_values` text,
+  `custom_fields_json` json,
```

Comment:
`custom_fields_json` should be in `glpi_assets_assets` table.
It doesn't make sense in this table (or I miss something).

Reply:
Fixed. I always manually create tables/new columns in my DB and then later change the GLPI SQL file/migrations. I copy-pasted into the wrong table.